### DISCUSSION TITLE
Advanced Downstream Analysis with new mudata object

### DISCUSSION THREAD

**Discussion by cedmo001** [aertslab/scenicplus#407] [General]
Hello! 

Thanks for creating such an amazing tool! Just wondering when the downstream analysis for the new pipeline will be available? 

Best Wishes, 
C

**Comment by SeppeDeWinter** [comment]
Hi @cedmo001 

Currently I'm quite busy (hence the late reply ;) ), so I'm not sure when this will be finished.

However, in the mean time you can convert your output to the old SCENIC+ object which you can use for all downstream analysis.

See: "Converting mudata output to old-style SCENIC+ object." in the tutorial: https://scenicplus.readthedocs.io/en/latest/human_cerebellum.html#Converting-mudata-output-to-old-style-SCENIC+-object.

Best,

Seppe

**Comment by cedmo001** [comment]
Hi Seppe, 

Thanks for the reply!

Do you have a recommendation for a downstream analysis tutorial I should follow? Was trying to use the old 10x PBMC multiome tutorial but it seems like the converted mudata to scplus_obj is missing some of the data that was originally stored in the unstructured data, so some of the functions don't seem to work? Unless I'm missing something? 

Best Wishes, 
C

**Reply by SeppeDeWinter** [reply]
Hi @cedmo001 

You are correct. I still have to make some updates here and provide tutorials (work in progress).

Best,

Seppe

**Reply by uqnsarke** [reply]
Do you have updates in this context?

**Reply by ccruizm** [reply]
Also interested in this 🙏🏼

**Reply by cedmo001** [reply]
Hi @SeppeDeWinter, 

Do you have any updates or and ETA on when some more advanced downstream analysis tutorials with be available. The old pipeline/tutorial had so much versatility in what you could do with the data, and just hoping to do some similar things with the new snakelike pipeline. Thanks again for such a wonderful tool, just everything updated in the middle of me running the old analysis/tutorial, and was hoping to try and complete more of this analysis in the new year! 

Best Wishes, 
C

**Reply by SeppeDeWinter** [reply]
Hi @cedmo001 

Are there particular functionalities you are missing at the moment?
I recently implemented the perturbation simulation: https://scenicplus.readthedocs.io/en/latest/Perturbation_simulation.html

Best,

Seppe

**Reply by wgao688** [reply]
Hi @SeppeDeWinter , where do we access the codebase for working with the "old-style" SCENIC+ object after the "Converting mudata output to old-style SCENIC+ object" section? It seems like that was able to produce figures like this. Thanks for your help!

![Screenshot 2025-01-13 at 8 29 39 AM](https://github.com/user-attachments/assets/7c17e4b2-5316-4469-9a60-6790d7cb8944)

**Reply by SeppeDeWinter** [reply]
Hi @wgao688 

I'm afraid to say that this figure was drawn using cytoscape. We never head function to automatically draw such figures.

Best,

Seppe

**Reply by cedmo001** [reply]
Hi @SeppeDeWinter,

I guess I'm struggling with filtering only the (+/+) and (-/+) regulons, as this isn't included in the new tutorial, and can't seem to figure out how to do this on my own (python isn't my forte haha). As well, I would like to enrich for eregulons that have a correlation with gene expression and AUC values, and then be able to view these eregulons on the dot plot. As of now I can't figure out how to have the color modality in the dot plot represent the gene expression values and the size modality represent my filtered eregulons. Apologies if these are really easy fixes, but just can't seem to get it to work. Thank you!

Best, 
C

**Reply by wgao688** [reply]
Hi @SeppeDeWinter, similarly I am dealing with the issue of having too many eRegulons and trying to narrow on some "stronger" candidates. Do you have any suggestions regarding filtering eRegulons, based for example on AUC, RSS, or other metrics? Also would you recommend focusing on "+/+" interactions out of the 4 possibilities since activators are more reliably identified than repressors?
